### PR TITLE
add section on OGC API - Maps (#90)

### LINKED ISSUE #90: add section on OGC API - Maps

### PR DESCRIPTION
**Pull request by tomkralidis**
Fixes #90

### CODE REVIEW

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 1]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
 # Exercise 5 - Maps of geospatial data via OGC API - Maps
 
 TODO
+
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:8]: remove the word "TODO"

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 2]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
...
+
+[OGC API - Maps](https://ogcapi.ogc.org/maps) provides a Web API to access
+any geospatial data as a georefenced map image.
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:11]: georefenced --> georeferenced

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 3]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
...
+
+pygeoapi supports the OGC API - Maps draft specification, using [MapServer MapScript](https://www.mapserver.org/mapscript) and a WMS facade as core backends.
+
+!!! note
+
+    See [the official documentation](https://docs.pygeoapi.io/en/latest/data-publishing/ogcapi-maps.html) for more information on supported map backends
+
+## Serve a WMS via OGC API - Maps
+
+Let's use pygeoapi's WMSFacade provider as a bride to serve an OGC WMS via OGC API - Maps.
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:25]: bride --> bridge (?)

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 4]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
...
+
+We can use the MapServer demo server at https://demo.mapserver.org/cgi-bin/msautotest
+
+!!! note
+
+    Feel free to use the WMS of your choice as you wish!
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:31]: the WMS of your choice --> a WMS of your choice

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 5]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
...
+
+!!! question "Update the pygeoapi configuration"
+
+    Open the pygeoapi configuration file in a text editor. Add a new dataset section as follows:
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:35]: See Features section: thought we decided to use uncommenting. So something like
*...scroll to the line "## START - EXERCISE 5 - Maps", uncomment or paste the configuration snippet below until the line that reads "## END - EXERCISE 5 - Maps". Be sure to keep the proper YAML indentation.*

File: workshop/content/docs/publishing/ogcapi-maps.md [hunk 6]
```diff
@@ -5,3 +5,104 @@ title: Exercise 5 - Maps of geospatial data via OGC API - Maps
...
+    <div class="termy">
+    ```python
+    >>> from owslib.ogcapi.maps import Maps
+    >>> m = Maps('http://localhost:5000')
+    >>> data = m.map('wms-facade-demo', width=1200, height=800, transparent=False)
+    >>> with open("output.png", "wb") as fh:
+    ...     fh.write(data.getbuffer())
+    ```
+    </div>
+
```
Comment (justb4) [inline workshop/content/docs/publishing/ogcapi-maps.md:102]: Don't know if we should mention, but as an alternative one could bash into the running pygeoapi container:
`docker exec -it pygeoapi /bin/bash`
then invoke `python3`

use the same OWSLib lines (OWSLib is already installed), but only `m = Maps('http://localhost:80')`. etc.

**Review by justb4 — CHANGES_REQUESTED** [review]
Some single line comments.

One main: docs/publishing/index.md needs updating and renumbering as well to insert 'Maps' chapter/.md

**Comment by justb4** [comment]
Ha, was still reviewing when you were changing!

File: workshop/content/docs/publishing/ogcapi-maps.md (hunk 6 above)
Comment (tomkralidis) [reply workshop/content/docs/publishing/ogcapi-maps.md:102]: Pehaps we can keep as is for now to be consistent with other sections?

Comment (justb4) [reply workshop/content/docs/publishing/ogcapi-maps.md:102]: Ok.

**Review by justb4 — CHANGES_REQUESTED** [review]
All GTG except docs/publishing/index.md (see my comment) to insert maps-file and renumber.

**Comment by tomkralidis** [comment]
> All GTG except docs/publishing/index.md (see my comment) to insert maps-file and renumber.

Done/updated.

**Comment by tomkralidis** [comment]
> All GTG except docs/publishing/index.md (see my comment) to insert maps-file and renumber.

Collaboration :)

**Pull request merged**